src/chapters/interfaces-and-polymorphism/exercises.rst

```diff
@@ -0,0 +1,176 @@
+Exercises: Interfaces and Polymorphism
+=======================================
+
+As a new C# coder, it might take you some time to recognize when to use interfaces.
+
+To help overcome this, let's consider a common occurrence---sorting a
+``List`` of objects.
+
+#. If the list contains ``string`` or numerical entries, then sorting the list
+   is :ref:`trivial <listsort>`:
+
+   .. sourcecode:: csharp
+
+      listName.Sort();
+
+#. However, if the elements are custom objects (like ``Cat``), then sorting the
+   list becomes more complicated. This is because the objects may contain
+   multiple fields, any of which could be used as a sorting option. For
+   ``Cat`` objects, this could include ``name``, ``age``, or ``mass``.
+
+Getting Started
+---------------
+
+Fork and clone the `starter code <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn7interfaces>`_ and open up Visual Studio to get started!
```

Comment:
oh ok, so im officially going crazy. id set up a practice project to test the code blocks in the lesson 7 content and just didnt realize that i had done that for me only and not for the reader of the text 🤦‍♀and since the examples so closely rely on what classes we used in the lsn6 demos it feels redundant to ask them to pull down something else with similar starting code. im sorry please ignore this suggestion!

Reply:
No worries!